[nibeheatpump] Updated NibeGW readme (#16864)
openhab/openhab-addons: bundles/org.openhab.binding.nibeheatpump/contrib/NibeGW/Arduino/README.md

##  ProDiNo ESP32 Ethernet v1 

NibeGW default settings are valid for ProDiNo ESP32 Ethernet v1 board.
Dynamic configuration is enabled by default.

### Config.h:

> ```

Reviewer: Perhaps syntax highlighting could be better if language is provided?
```suggestion
```c
```

//#define PRODINO_BOARD
#define PRODINO_BOARD_ESP32
//#define TRANSPORT_ETH_ENC28J6A0
```

### NibeGW.h:

> ```

Reviewer: ```suggestion
```c
```

## Debugging

Debugging messages are available by connecting to port 23 to your NibeGW via telnet.
Enable debugging in Config.h:

```
#define ENABLE_DEBUG
#define VERBOSE_LEVEL 5
#define ENABLE_REMOTE_DEBUG     // Remote debug is available in telnet port 23
```

You can connect to NibeGW with any telnet client.
You can also set some options via telnet.

> With 'h' you get a menue with all available options:

Reviewer: ```suggestion
With 'h' you get a menu with all available options:
```

nc -lu 9999 | hexdump -C
```

> ## Dynamic configuration

Reviewer: ```suggestion
## Dynamic Configuration
```

> When dynamic configuration is enabled (only ESP32 boards), NibeGW can be configured via WiFi connection.

Reviewer: ```suggestion
When dynamic configuration is enabled (only ESP32 boards), NibeGW can be configured via Wi-Fi connection.
```

Also OTA firmware update is supported.

> Following libraries are required

Reviewer: ```suggestion
The following libraries are required:
```

 * Bleeper (tested with version 1.1.0)
 * ElegantOTA (tested with version 2.2.9)

Dynamic configuration mode is loaded if input 0 is ON during boot.

> When dynamic configuration mode is activated, login to the 'Bleeper' WiFi Access point.

Reviewer: ```suggestion
When dynamic configuration mode is activated, login to the 'Bleeper' Wi-Fi access point.
```

Configuration page is available on port 80.
OTA update page on port 8080.

> ### Disable dynamic configuration

Reviewer: ```suggestion
### Disable Dynamic Configuration
```

Dynamic configuration can be disabled by commenting out following line from config.h file.

> ```

Reviewer: ```suggestion
```c
```
